### DISCUSSION TITLE
React Chart Library 선정

### DISCUSSION THREAD

**Discussion by seongpil0948** [seongpil0948/framework-next#46] [Polls]
[top-react-chart-libraries](https://ably.com/blog/top-react-chart-libraries)
기반 라이브러리 선정안

- [D3](https://observablehq.com/@d3/gallery?utm_source=d3js-org&utm_medium=nav&utm_campaign=try-observable)
- [Google Charts](https://www.react-google-charts.com/examples)
- [Apache](https://echarts.apache.org/examples/en/index.html)
- [ChartJs](https://www.chartjs.org/docs/latest/samples/information.html)

기한 : 다음주 월요일

**Comment by emirlan0706** [comment]
kill option while router provider

**Comment by seongpil0948** [comment]
구글차트로 결정.

**Comment by Aghaei-Dev** [comment]
use ant design charts 
easy configuration and a lot of charts kind 
https://ant-design-charts.antgroup.com
